caja.pot

```diff
@@ -6,9 +6,9 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: MATE Desktop Environment\n"
```

Comment:
PACKAGE VERSION is wrong, this should always be 
`"Project-Id-Version: MATE Desktop Environment\n"`

Project-Id-Version at Transifex is MATE Desktop Environment, so this need to be correct by hand after running ./makepot.
I will change this here.

Reply:
OK, Thank you.